docs/domains/KnowledgeArtifactManagement/services/TerminologyService/openapi.yml

```
@@ -0,0 +1,540 @@
+    Bearer:
+      type: http
+      description: Authorization using JWT
+      scheme: Bearer
+      bearerFormat: JWT
+security:
+  - Bearer: [ ]
+tags:
+  - name: Config
+    description: Configuration
+  - name: Fhir
+    description: FHIR Terminology Operations
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)